### PR TITLE
[a11y] Fixes contrast in selection foreground color in Exception Caug…

### PR DESCRIPTION
**Pull request by netonjm**
Fixes contrast in selection foreground color in Exception Caught window

Fixes VSTS #648206 - Luminosity contrast ratio for the white color text (with grey color
 background) in "Exception Caught" window is not matching the standard ratio

https://devdiv.visualstudio.com/DevDiv/_workitems/edit/648206

Now we match colors with other views in both themes dark/light

colors-black
colors-white

### CODE REVIEW

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs [hunk 1]
```diff
@@ -777,38 +779,37 @@ string GetMethodMarkup (bool selected)
...
+			if (string.IsNullOrEmpty (foregroundColor)) {
+				return markup;
 			}
-
-			return markup;
+			return $"<span foreground='{foregroundColor}'>{markup}</span>";
 		}
 
-		string GetFileMarkup (bool selected)
+		string GetFileMarkup (bool selected, bool focused, string foregroundColor)
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs]: focused is unused?

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs [hunk 2]
```diff
@@ -765,7 +767,7 @@ public StackFrameCellRenderer (Pango.Context ctx)
 			Context = ctx;
 		}
 
-		string GetMethodMarkup (bool selected)
+		string GetMethodMarkup (bool selected, bool focused, string foregroundColor)
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs]: focused is unused.

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/Styles.cs [hunk 3]
```diff
@@ -31,6 +31,8 @@ namespace MonoDevelop.Debugger
 {
 	public static class Styles
 	{
+		const string ObjectValueTreeValueSelectedText = "#FFFFFF";
+
```
Comment (rodrmoya) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/Styles.cs]: shouldn't this have a light/dark variants like the other colors -> https://github.com/mono/monodevelop/blob/34b459479ed276a66b0062f110924824794a6d06/main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/Styles.cs#L77 ?

Comment (sevoku) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/Styles.cs]: No, but please use `Styles.BaseSelectionTextColor` (which takes theme into account).

**Comment by netonjm** [comment]
@sevoku done!

**Pull request merged**